common/script/mapping-tables.js

```diff
@@ -204,6 +178,12 @@ function mappingTables() {
       detailsHTML += '</tbody></table></details>';
       details.innerHTML = detailsHTML;
 
+      var selfLink = document.createElement('a');
+      selfLink.className = "self-link";
+      selfLink.href = "#" + id;
+      selfLink.setAttribute('aria-label', 'permie');
```

Comment:
Maybe I'm missing some context here, but what is this? It seems we'll be left with a bunch of links with identical accNames of "permie"?

Reply:
Whoops, this was accidentally left in, I was testing adding a self link, thanks for catching!